packages/block-editor/src/hooks/font-size.js

```diff
@@ -60,10 +61,11 @@ function addSaveProps( props, blockType, attributes ) {
 		return props;
 	}
 
-	const { fontSize } = attributes;
+	// Use TokenList to dedupe classes.
+	const classes = new TokenList( props.className );
+	classes.add( getFontSizeClass( attributes.fontSize ) );
+	const newClassName = classnames( classes.value );
```

Comment:
do we really need `classnames` call now?

Reply:
nah, we don't https://github.com/WordPress/gutenberg/pull/21153/commits/652e85b9b22e932bdf17d36ef5639da88e1180b4